circle.yml

```diff
@@ -1,6 +1,16 @@
 version: 2
 jobs:
 
+  tsan:
+    macos:
+      xcode: "9.0"
+    steps:
+      - checkout
+      # Pre-cache
+      - run: swift run -Xswiftc -sanitize=thread swiftlint lint --lenient
```

Comment:
Should we use `--enable-all-rules`?

Reply:
Maybe 🤔. Individual rules really shouldn't be doing anything that would cause thread sanitizer violations though.